src/Resources/config/persister.xml

```diff
@@ -14,6 +14,14 @@
             <tag name="fos_elastica.pager_persister" persisterName="in_place" />
         </service>
 
+        <service id="fos_elastica.async_pager_persister" class="FOS\ElasticaBundle\Persister\AsyncPagerPersister" public="true">
```

Comment:
Any reason why this service needs to be public?

Reply:
I've inherited this definition by @rimas-kudelis. It depends if this service could be retrieved by container or it is only injected into services.

Reply:
@giovannialbero1992 the question was specifically about `public="true"`. You could try removing this attribute. I don't recall why I added it, but it could as well be copy-paste from the Enqueue bundle.

Reply:
Yes indeed, I do not see any reason why this service should be public.

Reply:
@rimas-kudelis Indeed I've answered to XWB about `public="true"` if it is defined, it permits to retrieve service by the container with a `$container->get('fos_elastica.async_pager_persister')` so if it is not necessary I'll remove it in a while